packages/dcos-launch/extra/launch/aws.py

```diff
@@ -50,6 +48,8 @@ def zen_helper(self, config):
         will be provided (must be done in correct order) and added to the info
         JSON as 'temp_resources'
         """
+        if config['zen_helper'] != 'true':
+            return {}
```

Comment:
How likely is this method to be called and this condition being met? Would it be better to throw an error here?